README.md

```diff
@@ -94,6 +95,7 @@ This file contains a number of front-end interview questions that can be used wh
 * How is responsive design different from adaptive design?
 * Have you ever worked with retina graphics? If so, when and what techniques did you use?
 * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
+* What is the difference between these two selectors 1) .one .two {} 2) .one.two {}
```

Comment:
Yes. I agree with your opinion.

Andrii
________________________________
Von: Arthur Verschaeve<mailto:notifications@github.com>
Gesendet: ‎16.‎05.‎2015 13:44
An: h5bp/Front-end-Developer-Interview-Questions<mailto:Front-end-Developer-Interview-Questions@noreply.github.com>
Cc: Andrii Malaman<mailto:malaman@outlook.com>
Betreff: Re: [Front-end-Developer-Interview-Questions] Add questions regarding CORS, CSS Selectors, JS Debug and JS loops (#296)

> @@ -94,6 +95,7 @@ This file contains a number of front-end interview questions that can be used wh
>  * How is responsive design different from adaptive design?
>  * Have you ever worked with retina graphics? If so, when and what techniques did you use?
>  * Is there any reason you'd want to use `translate()` instead of *absolute positioning*, or vice-versa? And why?
> +* What is the difference between these two selectors 1) .one .two {} 2) .one.two {}

This seems like soo beginner that if somebody knows how to answer the other questions this one doesn't make much sense anymore, don't you think so?

---
Reply to this email directly or view it on GitHub:
https://github.com/h5bp/Front-end-Developer-Interview-Questions/pull/296/files#r30460199